src/main/java/com/dao/homework/dao/jdbc/ManufacturerJdbcDaoImpl.java

```diff
@@ -0,0 +1,122 @@
+package com.dao.homework.dao.jdbc;
+
+import com.dao.homework.dao.ManufacturerDao;
+import com.dao.homework.exceptions.DataProcessingException;
+import com.dao.homework.lib.Dao;
+import com.dao.homework.model.Manufacturer;
+import com.dao.homework.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerJdbcDaoImpl implements ManufacturerDao {
+    private static final String TABLE_NAME = "manufacturer";
+    private static final String MANUFACTURER_NAME = "manufacturer_name";
+    private static final String MANUFACTURER_COUNTRY = "manufacturer_country";
+    private static final String MANUFACTURER_ID = "manufacturer_id";
+    private static final String DELETED = "deleted";
```

Comment:
it is optional to create constants for table columns. But if you want, you can leave it as it is)